app/helpers/session_timeout_warning_helper.rb

```diff
@@ -23,6 +23,7 @@ def auto_session_timeout_js
       render partial: 'session_timeout/ping',
              formats: [:js],
              locals: {
+               timeout_path: timeout_url,
```

Comment:
Nit: A bit odd we call this `timeout_path`, when we pass the absolute URL, and when `timeout_path` URL helper does exist and returns a different value.

```
[2] pry(main)> timeout_url
=> "http://localhost:3000/timeout"
[3] pry(main)> timeout_path
=> "/timeout"
```

Reply:
good catch, renamed to `timeout_url` consistently in b5a687f